tests/availability/owner.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+set -e
+
+CUR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $CUR/../_utils/test_prepare
+WORK_DIR=$OUT_DIR/$TEST_NAME
+CDC_BINARY=cdc.test
+
+MAX_RETRIES=10
+
+function test_owner_ha() {
+    test_kill_owner
+    test_hang_up_owner
+    test_expire_owner
+}
+# test_kill_owner starts two captures and kill the owner
+# we expect the live capture will be elected as the new
+# owner
+function test_kill_owner() {
+    echo "run test case test_kill_owner"
+    # start a capture server
+    run_cdc_server $WORK_DIR $CDC_BINARY
+    # ensure the server become the owner
+    ensure $MAX_RETRIES "$CDC_BINARY cli capture list 2>&1 | grep '\"is-owner\": true'"
+    owner_pid=$(pidof $CDC_BINARY)
+    owner_id=$($CDC_BINARY cli capture list 2>&1 | awk -F '"' '/id/{print $4}')
+    echo "owner pid:" $owner_pid
+    echo "owner id" $owner_id
+
+    # run another server
+    run_cdc_server $WORK_DIR $CDC_BINARY
+    capture_id=$($CDC_BINARY cli capture list 2>&1 | awk -F '"' '/id/{print $4}' | grep -v "$owner_id")
+    echo "capture_id:" $capture_id
+
+    # kill the server
+    kill $owner_pid
+
+    # check that the new owner is elected
+    ensure $MAX_RETRIES "$CDC_BINARY cli capture list 2>&1 |grep $capture_id -A1 | grep '\"is-owner\": true'"
+    echo "test_kill_owner: pass"
+
+    cleanup_process $CDC_BINARY
+}
+
+# test_hang_up_owner starts two captures and stops the owner
+# by sending a SIGSTOP signal.
+# We expect another capture will be elected as the new owner
+function test_hang_up_owner() {
+    echo "run test case test_hang_up_owner"
+
+    run_cdc_server $WORK_DIR $CDC_BINARY
+    # ensure the server become the owner
+    ensure $MAX_RETRIES "$CDC_BINARY cli capture list 2>&1 | grep '\"is-owner\": true'"
+
+    owner_pid=$(pidof $CDC_BINARY)
```

Comment:
`
[2020-04-02T14:29:07.788Z] /home/jenkins/agent/workspace/cdc_ghpr_integration_test/go/src/github.com/pingcap/ticdc/tests/availability/owner.sh: line 26: pidof: command not found`. [CI record](https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/cdc_ghpr_integration_test/detail/cdc_ghpr_integration_test/618/pipeline/103)
The CI environment may be too old 🙁️